lib/ansible/modules/cloud/spotinst/spotinst_aws_elastigroup.py

```diff
@@ -311,7 +311,8 @@
         In case of update it will override the existing Security Group with the new given array
     required: true
 
-  shutdown_script:
+  shut_down_script:
```

Comment:
This mismatches the actual arg, which is `shutdown_script` https://github.com/ansible/ansible/pull/36425/files#diff-1e53c41e763165b41238f29d4aa81cbcL1439